aep/general/0137/aep.md.j2

```diff
@@ -0,0 +1,130 @@
+# Apply
+
+In REST APIs, it is customary to make a `PUT` request to a collection's URI (for
+example, `/v1/publishers/{publisher}/books`) in order to create or replace a new
+resource within that collection.
+
+[Resource-oriented design](/resources) honors this pattern through the `Apply`
+method. These RPCs accept the parent collection and the resource to create or
+replace, and then return, the final resource.
+
+## Guidance
+
+APIs **should** provide an apply method for resources unless it is not valuable
+for users to do so.
```

Comment:
This seriously deviates from HTTP, in which PUT against a collection would be interpreted as a request to replace it in entirety (while the request to create a resource within a collection would be POST against the collection or PUT against the URL of the child-to-be [in cases where such a URL is client-provided]). What motivates the divergence?

Reply:
Apologies, I think this was a bad copy-paste when I copied this from create.

The intention is for this to describe a pattern to do a `PUT` against the resource - thereby replacing it. There is no intention to describe a collection-level PUT at the moment. 

I agree with everything you said, I'll fix the typo.

Reply:
the latest commit fixes the typo. Can you review the new wording?